internal/pgengine/engine.go

```diff
@@ -102,27 +97,28 @@ func StartEngineUsingPgDir(pgDir string) (pgEngine *Engine, retErr error) {
 		return nil, err
 	}
 
-	pgEngine = &Engine{
+	pgEngine := &Engine{
+		superuser: defaultSuperuser,
+
 		dbPath:   dbPath,
 		sockPath: sockPath,
-		user:     currentUser,
 		process:  process,
 	}
 	defer func() {
 		if retErr != nil {
 			pgEngine.Close()
 		}
 	}()
-	if err := pgEngine.waitTillServingTraffic(maxConnAttemptsAtStartup, waitBetweenStartupConnAttempt); err != nil {
+	if err := pgEngine.waitTillServingTraffic(defaultMaxConnAttemptsAtStartup, defaultWaitBetweenStartupConnAttempt); err != nil {
 		return nil, fmt.Errorf("waiting till server can serve traffic: %w", err)
 	}
 
 	return pgEngine, nil
 }
 
-func initDB(currentUser *user.User, initDbPath, dbPath string) error {
+func initDB(initDbPath, dbPath string, superuser string) error {
```

Comment:
(Nit.) If they're all string we can just do `(x, y, z string)`?

Reply:
ahhh true. This is a weird middle ground!